bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/client/AbstractKNXClient.java

```diff
@@ -200,11 +192,11 @@ private synchronized boolean connect() {
                     TimeUnit.MILLISECONDS);
 
             statusUpdateCallback.updateStatus(ThingStatus.ONLINE);
-            cancelReconnectJob(false);
             return true;
         } catch (KNXException | InterruptedException e) {
             logger.debug("Error connecting to the bus: {}", e.getMessage(), e);
             disconnect(e);
+            knxScheduler.schedule(this::connect, autoReconnectPeriod, TimeUnit.SECONDS);
```

Comment:
Shouldn't this better simply be a call of `scheduleReconnectJob()`, so that the field `connectJob` is correctly set?